README.md

```diff
@@ -166,9 +166,13 @@ Next, you need to make a file `/etc/cvmfs/default.local` manually; this file is
 contains, for instance, the URL to your local proxy and the size of the local cache. As an example, you can put
 the following in this file, which corresponds to not using a proxy and setting the local quota limit to 40000MB:
 ```
-CVMFS_HTTP_PROXY=DIRECT
+CVMFS_CLIENT_PROFILE=single
```

Comment:
@bedroge Can you open an issue to remind ourselves to properly document why we do this, with pointers to the CVMFS docs?

Reply:
Done! #77

Reply:
see #77